examples/2d/text2d.rs

```diff
@@ -1,3 +1,10 @@
+//! Example to show text rendering with moving, rotating and scaling text.
```

Comment:
```suggestion
//! Shows text rendering with moving, rotating and scaling text.
```

Saying it is an example doesn't convey any new or useful information.